.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Repeating from other reviews: Ned has suggested that we run the tests on all PRs, not just the ones targeting master.  Discussion at https://edx-internal.slack.com/archives/CD89W7Z7F/p1606849364465500